### DISCUSSION TITLE
Can't view anything other than text

### DISCUSSION THREAD

**Discussion by mebelsduggan** [ReagentX/imessage-exporter#179] [Q&A] [answered]
Disclaimer--complete noob to terminal commands & programming more generally, esp in the iOS/Mac ecosystem

Running MacOS 13.5.1, just downloaded all my text filed to html. When I open in firefox, I can't display anything other than texts, timestamps, read receipts, etc. Is this expected? What do I need to do to save images, videos, etc as well?

Thx

**Comment by ReagentX** [comment, marked as answer]
From the help text:

```
-c, --copy-method <compatible, efficient, disabled>
        Specify a method to use when copying message attachments
        Compatible will convert HEIC files to JPEG
        Efficient will copy files without converting anything
        If omitted, the default is `disabled`
```

For more info, see: https://github.com/ReagentX/imessage-exporter/tree/develop/imessage-exporter#examples
